exercises/practice/reverse-string/.approaches/performance/content.md

```diff
@@ -0,0 +1,109 @@
+# Performance
+
+On the [approaches page][approaches], we described several approaches:
+
+1. Using LINQ
+2. Using `Array.Reverse()`
+3. Using a `StringBuilder`
+
+In this document, we'll find out which of these approaches is the most performant one.
```

Comment:
I don't love "document" here but I don't have a better suggestion.

Reply:
I've restructured things a bit

Reply:
I think `Span<T>` is really its own approach, and the `performance` document should link to it.